### PR TITLE
use materialized view to generate rolling average block time

### PR DESCRIPTION
**Pull request by ShiningRay**
The previous way to generate rolling average block time statistics is manually aggregate data from database using rails code.
Now I take advantage of Materialized View to generate statistics from the database level, so that the source codes can be simplified.
And the ancient data can be rebuilt directly.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -418,6 +418,46 @@
...
+    t.integer "to_id"
+    t.integer "transaction_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["from_id"], name: "index_token_transfers_on_from_id"
+    t.index ["item_id"], name: "index_token_transfers_on_item_id"
+    t.index ["to_id"], name: "index_token_transfers_on_to_id"
+    t.index ["transaction_id"], name: "index_token_transfers_on_transaction_id"
+  end
+
```
Comment (Keith-CY) [inline db/schema.rb:460]: These codes are not related to this issue

File: test/services/charts/daily_statistic_generator_test.rb [hunk 2]
```diff
@@ -35,12 +35,5 @@ class DailyStatisticGeneratorTest < ActiveSupport::TestCase
       end
     end
 
-    test "average_block_time should return 840 points" do
```
Comment (Keith-CY) [inline test/services/charts/daily_statistic_generator_test.rb:38]: Could we add tests of the materialized view?

File: db/schema.rb (hunk 1 above)
Comment (zmcNotafraid) [reply db/schema.rb:460]: +1

**Review by zmcNotafraid — APPROVED** [review]

**Comment by Keith-CY** [comment]
Conflicted

**Review by Keith-CY — APPROVED** [review]
